test/shape_test.cpp

```diff
@@ -737,6 +738,22 @@ TEST_CASE(tuple)
     EXPECT(test::throws([&] { s.visit_type([](auto) {}); }));
 }
 
+TEST_CASE(return_shape_tuple)
+{
+    using migraphx::shape;
```

Comment:
replace `shape::` with `migraphx::shape` in callsites.  it is okay to use `using` but alteast here `shape::` is only used in couple of places.

Reply:
`    std::vector<shape> s{{shape::float_type}, {shape::float_type}};`

This line looks ugly, with three times the shape scope expansion.. Overflows to 2nd line. 
` std::vector<migraphx::shape> s{{migraphx::shape::float_type}, {migraphx::shape::float_type}};`

That is why `using` is very useful here for readability.